helm/charts/kratos/Chart.yaml

```diff
@@ -3,5 +3,5 @@ appVersion: v0.9.0-alpha.2
 description: A ORY Kratos Helm chart for Kubernetes
 name: kratos
 icon: https://raw.githubusercontent.com/ory/docs/master/docs/static/img/logo-kratos.svg
-version: 0.23.2
+version: 0.23.3
```

Comment:
This is auto-generated during release, please do not modify it manually 😉

Reply:
oh that's right. Done.